.github/actions/spelling/expect.txt

```diff
@@ -93,5 +93,6 @@ VNIC
 Vtpm
 vulnerabilityassessmentsettings
 vulnmgmt
+whois
```

Comment:
Can we stick this one in the spellcheck repo actually since that's just a generic command spelled correctly

Reply:
https://github.com/mondoohq/spellcheck-dictionary/pull/58

Reply:
@tas50 ready for review so that this PR turns green